src/get-started/localisation/index.md.njk

```diff
@@ -0,0 +1,92 @@
+---
+layout: layout-pane.njk
+title: Using localisation with GOV.UK Frontend
+section: Get started
+theme: How to guides
+order: 5
+description: Localisation allows you to translate GOV.UK Frontend components into other languages
+---
+
+Localisation allows you to translate GOV.UK Frontend components into other languages
+
+## Localisation using Nunjucks
+
+If you’re using the Nunjucks macros provided by GOV.UK Frontend, you can localise the UI text that appears within components by passing translation strings as named parameters.
+
+We have provided specific localisation guidance for some of the components. These include:
+
+* [accordion](/components/accordion/)
+* [character count](/components/character-count/)
+* [error message](/components/error-message/)
+* [footer](/components/footer/)
+* [header](/components/header/)
+* [summary list](/components/summary-list/)
+
+## Localisation using Javascript
+
+You can only pass translation strings to JavaScript with a list of key-value pairs.
+
+
+You can also pass a `locale` property containing a [two-letter ISO 639-1 code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes), which determines which plural forms and number formatting rules you use. If you do not include this, the `locale` defaults to the document language specified on the `html` element.
```

Comment:
We had some feedback from users who found this link confusing and overwhelming. This may be because they were trying out localisation in an artificial scenario (with no help from language-experts or translators). We could consider how we might help users with this, or publish as-is and see if we get any feedback.

Reply:
We should also be explicit about what the `locale` is used for and what it isn't used for, i.e: no automatic language switching.